modules/combined/test/tests/sliding_block/in_and_out/dirac/frictional_02_penalty.i

```diff
@@ -138,29 +138,43 @@
 
 [Materials]
   [./left]
-    type = LinearIsotropicMaterial
-    block = 1
-    disp_y = disp_y
-    disp_x = disp_x
-    poissons_ratio = 0.3
+    type = ComputeIsotropicElasticityTensor
+    block = '1'
     youngs_modulus = 1e6
+    poissons_ratio = 0.3
+  [../]
+  [./left_strain]
+    type= ComputeSmallStrain
```

Comment:
This should use ComputeFiniteStrain.  We should go back and make the SM cases use finite strain as well.